suite-native/atoms/src/Sheet/BottomSheet.tsx

```diff
@@ -25,18 +18,26 @@ const sheetWrapperStyle = prepareNativeStyle<WrapperStyleProps>((utils, { insetB
     borderTopLeftRadius: utils.borders.radii.large,
     borderTopRightRadius: utils.borders.radii.large,
     paddingBottom: Math.max(insetBottom, utils.spacings.medium),
-    height: 600,
+    height: 'auto',
 }));
 
 const sheetWithOverlayStyle = prepareNativeStyle(_ => ({
     flex: 1,
     justifyContent: 'flex-end',
 }));
 
+type BottomSheetProps = {
+    isVisible: boolean;
+    onVisibilityChange: (isVisible: boolean) => void;
+    children: ReactNode;
+    title?: string;
+    subtitle?: string;
+};
```

Comment:
props should stay on top of the file. Convention across all of suite-native packages. The idea is to have it easily accessible when you open component to see component API

Reply:
Ok, i reverted this. But I was following this style guide, which is recommending the arrangement I used: https://www.notion.so/satoshilabs/Components-9159b639d21b43ef8b66be150854b6b6

Please tell me if you follow the guide or not. Honestly, It makes sense to me that if in one file are two components (as in this file), then the props should be right above each of the components.

Reply:
Sorry for this we actually have little bit different convention in mobile vs desktop.

Reply:
Is it documented anywhere? If not i don't mind creating it. If you would me just a bullet list of the most important criteria, I can write them down. I believe it would be useful for those who will come later.